build.gradle.kts

```diff
@@ -30,26 +30,30 @@ repositories {
 dependencyLocking { lockAllConfigurations() }
 
 object Dependencies {
-  const val ecsLoggingVersion = "1.6.0"
-  const val openTelemetryVersion = "1.37.0"
+  const val ECS_LOGGING_VERSION = "1.6.0"
+  const val OPEN_TELEMETRY_VERSION = "1.37.0"
+  const val REDISSON_VERSION = "3.38.1"
+  const val MOCKITO_KOTLIN_VERSION = "5.2.1"
```

Comment:
are those modification related to this pr? 
I know that this is a little refactoring but this is out-of-scope into this pr.
making too much of this OOS modifications can lead to big pr's